apps/jobs/sender.py

```diff
@@ -55,27 +55,20 @@ def get_or_create_sqs_queue(queue_name):
     return queue
 
 
-def publish_submission_message(challenge_pk, phase_pk, submission_pk):
+def publish_submission_message(message):
     """
     Args:
-        challenge_pk: Challenge Id
-        phase_pk: Challenge Phase Id
-        submission_pk: Submission Id
+        message
```

Comment:
Can you please write its type and what all keys it contains.

Reply:
Added
```py
        message: A Dict with following keys
            - "challenge_pk": int
            - "phase_pk": int
            - "submission_pk": int
```